### PR TITLE
Build system and testing setup.

### PR DESCRIPTION
**Pull request by GoldenBadger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Arbiter
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/RazoftOSS/Arbiter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
\ No newline at end of file
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/RazoftOSS/Arbiter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (VoyTechnology) [inline README.md:3]: What happened that this line for removed and added again?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
+[![Build Status](http://cpssd1-web.computing.dcu.ie/buildStatus/icon?job=Arbiter Master)](http://cpssd1-web.computing.dcu.ie/job/Arbiter%20Master/)
```
Comment (VoyTechnology) [inline README.md]: Should there be a space between `Arbiter` and `Master`? I know its auto generated by Jenkins but it should be escaped with `%20`

**Comment by VoyTechnology** [comment]
Othen than the minor issues, LGTM

File: README.md (hunk 1 above)
Comment (GoldenBadger) [inline README.md:3]: No clue.

File: README.md (hunk 2 above)
Comment (GoldenBadger) [inline README.md]: Dunno, but it works. I'll change it anyway.

**Pull request merged**